modules/rosa-hcp-byo-oidc.adoc

```diff
@@ -29,11 +34,29 @@ This command returns the following information.
 +
 [source,terminal]
 ----
-I: This command will create a S3 bucket populating it with documents to be compliant with OIDC protocol. It will also create a Secret in Secrets Manager containing the private key.
-I: Please run command below to create a cluster with this oidc config:
-rosa create cluster --sts \
- --oidc-endpoint-url https://oidc-l8c0.s3.us-east-1.amazonaws.com \
- --oidc-private-key-secret-arn arn:aws:secretsmanager:us-east-1:269733383066:secret:rosa-private-key-oidc-l8c0-4vdMVv
+I: Setting up managed OIDC configuration
+I: Please run the following command to create a cluster with this oidc config
+rosa create cluster --sts --oidc-config-id 2330dbs0n8m3chkkr25gkkcd8pnj3lk2
+I: Creating OIDC provider using 'arn:aws:iam::42819244:user/user_Name'
+? Create the OIDC provider? Yes
+I: Created OIDC provider with ARN 'arn:aws:iam::42819244:oidc-provider/dvbwgdztaeq9o.cloudfront.net/2330dbs0n8m3chkkr25gkkcd8pnj3lk2'
 ----
 
-When creating your cluster, you must supply the OIDC endpoint URL and secret manager secret ARN. The CLI output provides both values for `--mode auto`, otherwise you must to determine these values based on `aws` CLI output for `--mode manual`.
\ No newline at end of file
+When creating your cluster, you must supply the OIDC endpoint URL and secret manager secret ARN. The CLI output provides both values for `--mode auto`, otherwise you must to determine these values based on `aws` CLI output for `--mode manual`.
```

Comment:
It doesn't need to set OIDC endpoint URL and secret manager secret ARN to cluster.
It need to set --oidc-config-id to cluster

Reply:
@yingzhanredhat @gdbranco @wgordon17 I removed the `--oidc-endpoint-url` references in favor of the `--oidc-config-id`. Please let me know if I missed any.